.github/actions/icr-build-and-push-images/action.yml

```diff
@@ -11,7 +11,7 @@ runs:
       uses: docker/build-push-action@v3
       with:
         context: .
-        file: ./infrastructure/docker/Dockerfile-ray-qiskit
+        file: ./client/Dockerfile-ray
```

Comment:
since we only have one Dockerfile in the client directory, do we still need the ray suffix?

Reply:
@IceKhan13 and @akihikokuroda happy to hear your opinions too here because I have several doubts with this.

Reply:
My 2 cents are: name the docker file Dockerfile-ray-node and place it in the repository root directory.

Reply:
I don't dislike that one, Aki 👍

Reply:
Done in [4a4c62d](https://github.com/Qiskit-Extensions/quantum-serverless/pull/868/commits/4a4c62df61b6b49ab91677e4b9da5332d92fc91e). I liked Aki approach but if you have better ideas let me know 😄